Core.Collectors/IO/AzureStorageResourceHelper.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.CloudMine.Core.Collectors.IO
+{
+    public static class AzureStorageResourceHelper
+    {
```

Comment:
Any reason not to have these in AzureHelpers.cs?

Reply:
moved it to StorageAccountHelpers